File: static-analyzer/test-snippets/LinearLoop.java
public class LinearLoop {
    public static int sum(int[] arr) {
        int total = 0;
        for (int i = 0; i < arr.length; i++) {
            total += arr[i];
        }
        return total;
    }

    public static void main(String[] args) {
        int[] data = {1, 2, 3, 4, 5};
        System.out.println(sum(data));
    }
}

File: static-analyzer/test-snippets-csharp/LinearLoop.cs
public class LinearLoop
{
    public static int Sum(int[] arr)
    {
        int total = 0;
        for (int i = 0; i < arr.Length; i++)
        {
            total += arr[i];
        }
        return total;
    }

    public static void Main()
    {
        int[] data = { 1, 2, 3, 4, 5 };
        System.Console.WriteLine(Sum(data));
    }
}
